docs/notifications.json

```
@@ -18,5 +13,10 @@
     "id": 79,
     "title": "<b>Influence the roadmap for 2024</b>",
     "text": "Take a few minutes to share your feedback and expectations in the <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-survey\" href=\"https://tally.so/r/3Ex4PN?source=docs-notification\">Developer Survey</a>."
+  },
+  {
+    "id": 80,
+    "title": "<b>MUI X v7 is now in beta</b>",
```

This might be better:

```suggestion
    "title": "<b>MUI X v7.0.0-beta.0</b>",
```